src/usr/local/www/system_crlmanager.php

```diff
@@ -148,6 +137,17 @@
 				$input_errors[] = gettext("Invalid serial in list (Must be ASN.1 integer compatible decimal or hex string).");
 			}
 		}
+		if (empty($pconfig['certref']) && empty($revoke_list)) {
```

Comment:
This should be after the block below it, shouldn't it? Otherwise `$revoke_list` will always be empty if they selected a certificate and didn't enter a serial number.

Reply:
now it check if(!empty($pconfig['revokeserial'])) and check each value for ASN.1 compatibility

working ok now